components/TheFooter.vue

```diff
@@ -97,7 +97,7 @@ const menu: Menu[] = [
   },
   {
     name: $i18n.t('faq.page'),
-    url: '/rmrk/faq',
+    url: '/faq',
```

Comment:
Sorry, as the review comment mentioned, we do not need to change the path for now.

Reply:
> Sorry, as the review comment mentioned, we do not need to change the path for now.

😆 Done